external/upstream/pybind11/CMakeLists.txt

```diff
@@ -11,7 +11,7 @@ else()
     include(ExternalProject)
     message(STATUS "Suitable pybind11 could not be located, ${Magenta}Building pybind11${ColourReset} instead.")
     ExternalProject_Add(pybind11_external
-        URL https://github.com/pybind/pybind11/archive/v2.2.3.tar.gz
+        URL https://github.com/pybind/pybind11/archive/v2.2.4.tar.gz
```

Comment:
for consistency, bump it [here](https://github.com/psi4/psi4/pull/1401/files#diff-e8cf7e78ee17d1826f8f9cef1a04c7bfR60), too, please.

Reply:
Done.